ui/ducks/send/send.js

```diff
@@ -603,7 +603,7 @@ export const initializeSendState = createAsyncThunk(
     // For instance, in the actions.js file we dispatch this action anytime the
     // chain changes.
     if (!draftTransaction) {
-      thunkApi.rejectWithValue(
+      return thunkApi.rejectWithValue(
         'draftTransaction not found, possibly not on send flow',
       );
```

Comment:
without returning this the rest of the code in this slice would run, this is unwanted. Returning here prevents attempts to access draftTransaction erroring out further in this code. Although in this case the error would be handled by redux and would not bubble up to sentry.